### PR TITLE
Support for Plex Movie Scanner

### PR DESCRIPTION
**Pull request by wampa1**
I think I figured out how to do a pull request. Forked it, then made the change then submit the request I guess.

This uses name and year matching to attempt to mitigate the plex movie scanner guid misalignment with Trakt's available lookups.

Hope this helps.  It definitely helped me (and worked like a champ).

Eric

### CODE REVIEW

**Comment by glensc** [comment]
@wampa1 in the future, create a branch when submitting PR. you may want your `master` to live it's own life, but it's directly linked to specific PR. any commits you add to branch linked to the PR, will add more commits to the PR. in other words, never create pull request from `master` branch.

File: main.py [hunk 1]
```diff
@@ -54,14 +54,31 @@ def process_movie_section(s, watched_set, ratings_dict, listutil, collection):
...
         else:
             logging.error('Movie [{} ({})]: Unrecognized GUID {}'.format(
                 movie.title, movie.year, movie.guid))
             continue
             raise NotImplementedError()
         # search and sync movie
         try:
-            search = trakt.sync.search_by_id(x, id_type=provider)
+            search = []
+            if ( provider == 'plex' ):
```
Comment (glensc) [inline main.py:72]:
```suggestion
            if provider == 'plex':

```

superfluous parens can be dropped

Comment (wampa1) [reply main.py:72]: If it's cool with you, I'd like to follow both of your suggestions.

I'll remove the parens (I am 99.99% a Java guy), kill this PR, and start a new one that's from a non-master branch.

I hope killing the PR and starting a new one doesn't cause you any additional concern.

Thanks again!

Comment (glensc) [reply main.py:72]: you can force push to the same branch you created the pr, and PR will update. no need to create new PR.

you can also apply the suggestion from web. there's UI for that. but you want then probably git squash the commit locally (and the force push).

**Pull request closed without merging**